cargo-miri/src/util.rs

```diff
@@ -88,6 +88,77 @@ pub fn escape_for_toml(s: &str) -> String {
     format!("\"{s}\"")
 }
 
+pub fn flagsplit(flags: &str) -> Vec<String> {
+    // This code is taken from `RUSTFLAGS` handling in cargo.
+    // Taken from miri-script util.rs
```

Comment:
That last part is not true, this function already existed here? You just moved it up. Why did you move it?

Reply:
An overside on my part, I'll put it back before definition of exec(), must have pressed d or x when in visual mode